src/components/pokemonCard/PokemonCard.js

```diff
@@ -0,0 +1,40 @@
+import React, {useEffect, useState} from "react";
+import axios from "axios";
+import './PokemonCard.css';
+function PokemonCard({name}) {
+
+    const [pokemonData, setPokemonData] = useState({});
+    useEffect(()=> {
+        async function fetchPokemon() {
+            try {
+                const result = await axios.get(`https://pokeapi.co/api/v2/pokemon/${name}`);
+                console.log(result);
+                setPokemonData(result);
+            } catch (e) {
+                console.error(e);
+            }
+        }
+        fetchPokemon();
+    }, []);
+    return (
+        <>
+            <section className="poke-card">
+            { Object.keys(pokemonData).length > 0 &&
```

Comment:
Dan kan deze check ook simpeler
```suggestion
            { pokemonData &&
```